pages/docs/balance/api-reference/open-banking/list-available-exchange-connections.mdx

```diff
@@ -0,0 +1,106 @@
+---
+layout: guides
+product: balance
+title: "List available exchange connections for a customer"
+guide:
+  step: 3
+meta:
+  title: "Dwolla API Reference | Open Banking: List available exchange connections"
+  description: "Retrieve a list of available exchange connections for a customer after they have completed the MX Connect widget flow. These connections represent external bank accounts that can be used to create funding sources for transfers."
+---
+
+# List available exchange connections for a customer
+
+This endpoint retrieves a list of a customer's external bank accounts that have been authorized through MX Connect. Each account is represented as an "available exchange connection" with details like the account name and associated `availableConnectionToken`. This information is essential for creating an exchange and corresponding funding source within Dwolla.
+
+<InlineCTA
+  icon={conceptsIcon}
+  text="Understand the MX integration flow. Read the Open Banking concept article which outlines the seqence of steps."
+  href="/docs/balance/open-banking"
+/>
+
+### HTTP request
+
+> `GET https://api.dwolla.com/customers/{id}/available-exchange-connections`
+
+### Request parameters
+
+| Parameter | Required | Type   | Description                          |
+| --------- | -------- | ------ | ------------------------------------ |
+| id        | yes      | string | Customer resource unique identifier. |
+
+### HTTP status and error codes
+
+| HTTP Status | Code     | Description                                                  |
+| ----------- | -------- | ------------------------------------------------------------ |
+| 200         | Ok       | The Dwolla API accepted the request and returned a response. |
+| 404         | NotFound | The requested resource was not found. Check Customer ID.     |
+
+### Request and response
+
+```bash
+GET https://api.dwolla.com/customers/1b54c81a-261f-4779-bb57-9405e6e00694/available-exchange-connections
+Accept: application/vnd.dwolla.v1.hal+json
+Authorization: Bearer pBA9fVDBEyYZCEsLf/wKehyh1RTpzjUj5KzIRfDi0wKTii7DqY
+
+...
+
+{
+    "_links": {
+        "self": {
+            "href": "https://api.dwolla.com/customers/1b54c81a-261f-4779-bb57-9405e6e00694/available-exchange-connections",
+            "type": "application/vnd.dwolla.v1.hal+json",
+            "resource-type": "customer"
+        },
+        "customers": {
+            "href": "https://api.dwolla.com/customers/1b54c81a-261f-4779-bb57-9405e6e00694",
+            "type": "application/vnd.dwolla.v1.hal+json",
+            "resource-type": "customer"
+        }
+    },
+    "_embedded": {
+        "available-exchange-connections": [
+            {
+                "availableConnectionToken": "eyJhY2NvdW50SWQiOiJBQ1QtMjAxY2FkM2MtYzc2Yi00N2M1LWI3Y2QtMTkxY2FhNzdlZWM5IiwibWVtYmVySWQiOiJNQlItZGNjZWY0ZWMtOGM4MC00NTlmLTlhMGItMTc1ZTA0OTJmZWIzIn0=",
+                "name": "Checking"
+            }
+        ]
```

Comment:
```suggestion
        "available-exchange-connections": [
            {
                "availableConnectionToken": "eyJhY2NvdW50SWQiOiJBQ1QtMjAxY2FkM2MtYzc2Yi00N2M1LWI3Y2QtMTkxY2FhNzdlZWM5IiwibWVtYmVySWQiOiJNQlItZGNjZWY0ZWMtOGM4MC00NTlmLTlhMGItMTc1ZTA0OTJmZWIzIn0=",
                "name": "XYZ Checking"
            },
            {
                "availableConnectionToken": "eyJhY2NvdW50SWQiOiJBQ1QtNWY1ZWIwYTgtYTJiNC00NGQxLTk3ODYtMWVmOTM5NzZkYTU4IiwibWVtYmVySWQiOiJNQlItNTcxN2E5MDQtNDA0NC00YjQyLWIxMDctZDQyOWE0YjIzOGY5In0=",
                "name": "ABC Savings"
            }
        ]
```

Thought adding one more item in the list would be helpful to show why this endpoint is helpful to have.